src/main/java/com/scalar/database/transaction/consensuscommit/CrudHandler.java

```diff
@@ -22,7 +22,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-/** */
+/** Perform CRUD operations on {@link DistributedStorage} */
```

Comment:
It looks some operations don't operate on DistributedStorage.

Reply:
I think it's misleading and not very informative. It should explain how it operates with snapshot and the storage.
Maybe, understanding snapshot isolation properly before writing a doc is good. 
Also, it's a comment for a class, please start with a noun.

Reply:
Thank you for the comments. I updated based on your feedback, so please check it when you have time.